Formula/kyma-cli.rb

```diff
@@ -19,6 +20,8 @@ def install
     ldflags = %W[
       -s -w
       -X github.com/kyma-project/cli/cmd/kyma/version.Version=#{version}
+      -X github.com/kyma-project/cli/cmd/kyma/install.DefaultKymaVersion=master
```

Comment:
```suggestion
      -X github.com/kyma-project/cli/cmd/kyma/install.DefaultKymaVersion=#{version}
```

Reply:
Master is for the binaries created from the master branch. For the releases, we're using the release version, which is the same as the `version` here.

Reply:
Sounds good, thanks for the catch! :)